jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_move_base/local_costmap_params.yaml

```diff
@@ -11,4 +11,5 @@ width: 5.0 # 3.0
 height: 5.0 # 3.0
 resolution: 0.025 # 0.06
 
-footprint: [[-0.325, -0.325], [-0.425, -0.175], [-0.425, 0.175], [-0.325, 0.325], [0.325, 0.325], [0.375, 0.0], [0.325, -0.325]]
+footprint: [[-0.34, -0.35], [-0.34, 0.35], [0.34, 0.35], [0.34, -0.35]]
```

Comment:
can you try `[[-0.34, -0.35], [-0.34, 0.35], [0.34, 0.35], [0.46, 0.0], [0.34, -0.35]]` tomorrow?
I thinks is is better to use same footprint in `safe_teleop_base`, too, probably.
https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_move_base/safe_teleop_pr2_params.yaml#L1